packages/client/components/EstimatePhaseDiscussionDrawer.tsx

```diff
@@ -26,6 +26,7 @@ const Drawer = styled('div')<{isDesktop: boolean; isOpen: boolean}>(({isDesktop,
   bottom: 0,
   top: 0,
   right: isDesktop ? 0 : undefined,
+  paddingBottom: isDesktop ? 0 : 56,
```

Comment:
I think hardcoding would not solve the problem and it also disturbs the design on mobile.

[Here](https://ishwar-rimal.medium.com/fixing-that-address-bar-issue-in-mobile-browsers-once-and-for-all-8c283fc88e56) is a post if it could help.

Reply:
I will look into this but meanwhile to test, you can forward your local development port to a live testing address with ngrok and access the address on mobile.
@87prashant

Reply:
Thanks for helping🙏

Reply:
After Researching this issue and trying out different suggested fixes of which none of them seems to work, I would highly suggest we go with this hard-coded fix as it seems to be the only thing that works (tested on both Chrome and Firefox on Android, See attached Video). It's also worth noting that there is no official / community agreed on a fix for this issue only workarounds based on different approaches which sometimes would involve modifying the CSS of the body of the webpage which is not visible in this case due to the fact that it will affect multiple pages in different ways. Based on this I highly suggest we go along with this fix as it addresses the issue. 

https://user-images.githubusercontent.com/31279142/184479188-f3821c76-45a6-41f4-93f2-47b51214be49.mp4

Reply:
@jesjo001 did you try setting the height to `100dvh`[^1]?

[^1]: https://dev.to/frehner/css-vh-dvh-lvh-svh-and-vw-units-27k4